x/metadata/keeper/msg_server.go

```diff
@@ -48,6 +59,44 @@ func (k msgServer) WriteScope(
 	return types.NewMsgWriteScopeResponse(msg.Scope.ScopeId), nil
 }
 
+func (k msgServer) createExpirationForScope(ctx sdk.Context, msg *types.MsgWriteScopeRequest) (*exptypes.Expiration, error) {
+	// create expire scope request.
+	expireMsg := types.NewMsgExpireScopeRequest(msg.Scope.ScopeId, msg.Signers)
+	wrapper, anyErr := codec.NewAnyWithValue(expireMsg)
+	if anyErr != nil {
+		return nil, anyErr
+	}
+
+	// use default expiration, but override with cmd line arg if provided
```

Comment:
Nit: We don't care about command line args in here. Most usage isn't through CLI anyway. There's a similar comment above (line 42 as of writing this). In here, we're just using values provided in the message.

Reply:
removed references to CLI from comment.